test/pg_regress.sh

```diff
@@ -194,4 +197,4 @@ mkdir -p ${EXE_DIR}/sql/dump
 export PG_REGRESS_DIFF_OPTS
 
 PG_REGRESS_OPTS="${PG_REGRESS_OPTS}  --schedule=${SCHEDULE}"
-${PG_REGRESS} "$@" ${PG_REGRESS_OPTS}
+${PG_REGRESS} "$@" ${PG_REGRESS_OPTS} 9>&2
```

Comment:
What is fd 9?

Reply:
I just wanted an fd to get my messages out to the console - which is not used by anyone - `9` looked high enough...
other end of the line is [here](https://github.com/kgyrtkirk/pg-dev-tools/blob/00d0dfb8494a89b9a0217020f9c7c4552fdea542/psql-debug/psql#L4-L5)

Reply:
Could you add a comment that explains the reason for the file descriptor to the file?

Reply:
Looking at this line, if you don't redirect the 9, it's going to write to 2 anyway? I'm asking because using a specific fd looks more like something for personal usage, not for generic scripts. I wonder if we could remove it while still enabling you to use it locally in a convenient way.

Reply:
stdout&stderr is captured by `pg_regress` [internally](https://github.com/postgres/postgres/blob/37a6d81c457fd507bac0a08758a5dcff5c6225f7/src/test/regress/pg_regress_main.c#L81)  and written to a file - if I write to `2` it will just sit there silently...hence the need to hack that around...

there are a few open fd-s so I've picked `9`
```
lr-x------ 1 dev dev 64 Jun 23 12:36 0 -> /home/dev/timescaledb/tsl/test/sql/chunk_api.sql
lrwx------ 1 dev dev 64 Jun 23 12:36 1 -> /dev/pts/2
l-wx------ 1 dev dev 64 Jun 23 12:36 2 -> 'pipe:[26009927]'
l-wx------ 1 dev dev 64 Jun 23 12:36 3 -> /home/dev/timescaledb/build/tsl/test/regression.out
lr-x------ 1 dev dev 64 Jun 23 12:36 4 -> /home/dev/timescaledb/test/temp_schedule
lr-x------ 1 dev dev 64 Jun 23 12:36 5 -> /proc/826754/fd
lrwx------ 1 dev dev 64 Jun 23 12:36 9 -> /dev/pts/2
```

...optionally I could also patch `pg_regress` to open `fd9`; but I would rather avoid that unless opening an fd is unacceptable

Reply:
@akuzm do you have a different approach in mind?

Reply:
@akuzm could you expand a little bit more what you had in mind?